### ISSUE TITLE
coding mistake

### ISSUE THREAD

**Issue by hanglin3** [LlionJ/blog#1]
According to the formula of the RMSProp, I think there is a little mistake in the define of g_rms() function.

def g_rms(gradients, state, learning_rate=0.1, decay_rate=0.99):
    if state is None:
        state = tf.zeros(DIMS)
    state = decay_rate*state + (1-decay_rate)*tf.pow(gradients, 2)
    **update = -learning_rate*gradients / (tf.sqrt(state)+1e-5)**
    return update, state

The highlight part should be modified to this:
update = -learning_rate*gradients / (tf.sqrt(state+1e-5))
